core/client/fs/src/main/java/alluxio/client/block/policy/SpecificHostPolicy.java

```diff
@@ -26,6 +26,9 @@
 /**
  * Always returns a worker with the specified hostname. Returns null if no active worker on that
  * hostname found.
+ *
+ * Note that since this does not have an empty constructor, this cannot
+ * be used as a default policy.
```

Comment:
Is this actually true? This is what the comments in `BlockLocationPolicy` said, but neither `LocalFirstPolicy` nor `LocalFirstAvoidEvictionPolicy` have empty constructors from what I saw.

Reply:
Good call. Can you try running locally with this policy? I don't think this policy is documented enough both on the purpose and how to use it.

Reply:
If this policy works, can you add a few lines (explaining how the configuration should look like) both in the code comment and the doc? From the code it seems this will make the allocation only go to the WORKER_HOSTNAME?